demo/src/index.re

```diff
@@ -28,6 +28,9 @@ module Link = [%styled.div {|
   margin-top: 16px;
 |}];
 
+module Empty = [%styled.span]
+module Empty1 = [%styled ""]
```

Comment:
Maybe adding those into "simple.ml" where you do the snapshotting as well would be a good idea.